iac/instance.py

```diff
@@ -140,14 +150,51 @@ def _ec2_config(
     return definition
 
 
+class Barrier:
```

Comment:
the default Barier doesn't do anything, so maybe this is a bad name.  Thoughts or suggestions?

Reply:
`NoOpBarrier`? Seems fine as is to me in general since it's pretty obvious it's a stub

Reply:
sounds good.  If it is obvious as is, I will just leave it.  Thank you for the input